spotipy/oauth2.py

```diff
@@ -505,10 +506,248 @@ def _add_custom_values_to_token_info(self, token_info):
         return token_info
 
 
+class SpotifyImplicitGrant(SpotifyAuthBase):
+    """ Implements Implicit Grant Flow for client apps
+
+    This auth manager enables *user and non-user* endpoints with only
+    a client secret, redirect uri, and username. The user will need to
+    copy and paste a URI from the browser every hour.
```

Comment:
what is the main reason that the ability to refresh the token without user input is sacrificed? Is there a blocker that makes it technically impossible currently or is it just not implemented, and can still be implemented in the future?

Reply:
The blocker is that the token comes back in a hash fragment (instead of a query string). I could not get the server to retrieve the fragment